kapprover/kapprover.go

```diff
@@ -73,10 +122,12 @@ func tryApprove(filters inspectors.Inspectors, deniers inspectors.Inspectors, wa
 		for _, filter := range filters {
 			message, err := filter.Inspector.Inspect(client, request)
 			if err != nil {
+				requestsError.WithLabelValues("filterError").Inc()
```

Comment:
Use the inspector.Name as the label value?

Reply:
Done